lib/cli.js

```diff
@@ -24,6 +25,8 @@ updateNotifier({ pkg }).notify();
 
 if (argv.v) {
   console.log(`v${pkg.version}`);
+} else if (!isGit()) {
```

Comment:
But in this case we would have two different conventions in one `if - else` statement

```js
if (statement) oneLiner();
else if (otherStatement) oneLiner();
else {
  notOneLiner();
}
```

The issue above has no real convention, both is valid.